pdocs/templates/text.mako

```diff
@@ -16,7 +16,18 @@ def ${func.name}(
     ${",\n    ".join(func.params())}
 )${returns}
 ```
+% if hasattr(func, 'parsed_docstring') and func.parsed_docsting:
+    # table with arguments info
+    % if func.parsed_docstring.params:
+| Parameter | type | description | default |
+|---|---|---|---|
+        % for p in func.parsed_docstring.params:
+| ${p.arg_name} | ${p.type_name} | ${p.description} | ${p.default} |
+        % endfor
+    % endif
```

Comment:
We can add the other attributes/sections of parsed_docstring here, i.e. raises and returns/yields

Reply:
Oh right, after the `%endif`, but before the `%else`